templates/config/overlays/namespaced/kustomization.yaml.tpl

```diff
@@ -0,0 +1,15 @@
+resources:
```

Comment:
I don't believe this needs to be a `.yaml.tpl` file since it contains no Go template injections. Renaming it to `.yaml` and adding it to the list of files that is copied when running `ack-generate release` ([adding it here](https://github.com/aws-controllers-k8s/code-generator/blob/e2a9338988f0f7c78c6073c959ff4994ef8394e9/pkg/generate/ack/release.go#L33)) should suffice.

Reply:
Sure doesn't need to be a .tpl file. Should it be a part of that ack-generate release however since all those other files pertain to helm/templates

Reply:
This does need to be a .tpl file because it contains template variables..